processingprocess: Add support for DataUrl

ptitjano opened the pull request:
For all the output layers, this allows to save the url of the associated file as DataUrl.

ptitjano committed:
processingprocess: Add support for DataUrl

For all the output layers, this allows to save the url of the associated
file as DataUrl.

Some tests are also added to test this new feature.

dmarteau commented:
Thanks.

Could you add a test for validating changes, please ? (you may issue request in tests with pyqgiswps.tests.HTTPTestCase)

ptitjano committed:
test_algorithms: Remove trailing whitespaces

ptitjano committed:
test_context: Remove trailing whitespaces

ptitjano committed:
test_layersio: Remove trailing whitespaces

ptitjano committed:
test_models: Remove trailing whitespaces

ptitjano commented:
> Thanks.
> 
> Could you add a test for validating changes, please ? (you may issue request in tests with pyqgiswps.tests.HTTPTestCase)

Thanks. I have added some tests... And fixed the existing ones that I had broken with the new "uuid" parameter in "run_algorihtm"
